### ISSUE TITLE
Still getting error when loading YTV in 9.249

### ISSUE THREAD

**Issue by ctbritt** [David-Zvekic/TokensVisible#30]
Here's the error message on world load: 
```
foundry.js:316 LibWrapperPackageError: Error detected in module Your Tokens Visible.

This is *not* a libWrapper bug.

Find information about this module here: https://github.com/David-Zvekic/TokensVisible

Ask the community for support:
- FoundryVTT Discord's #modules-troubleshooting channel: https://discord.gg/foundryvtt
- FoundryVTT Reddit: https://www.reddit.com/r/FoundryVTT

== Technical Details:
Detected by libWrapper.
Package ID= TokensVisible
Error= Can't wrap 'SightLayer._castRays', target does not exist or could not be found.

[Detected 1 package: TokensVisible]
    at new 🎁constructor (libWrapper-wrapper.js:111:11)
    at libWrapper-api.js:82:18
    at se (libWrapper-api.js:155:9)
    at Function.🎁register [as register] (libWrapper-api.js:522:14)
    at patch (TokensVisible.js:907:16)
    at Object.tokensVisible._setEnhancedCastRays (TokensVisible.js:775:9)
    at Object.tokensVisible.setProperCastRays (TokensVisible.js:113:37)
    at TokensVisible.js:94:19
    at Function._call (foundry.js:294:14)
    at Function.call (foundry.js:279:33)
    at Canvas.draw (foundry.js:19998:13)
    at async Scene.view (foundry.js:17740:7)
    at async Game.initializeCanvas (foundry.js:5676:20)
    at async Game.setupGame (foundry.js:5547:5)
    at async Game._initializeGameView (foundry.js:6498:5)
    at async Game.initialize (foundry.js:5430:5)
```

**Comment by David-Zvekic** [comment]
Fixed in 3.0.1

Even though enhanced raycasting is turned off by default in 3.0.0, the
setting was still stored in the world settings data if you used a prior
incompatible version of YTV and didn't uninstall it before installing 3.0.0.
 3.0.1 explicitly turns it off if it was previously set to 'Enhanced'.
Since none of the prior versions officially supported v9 I didn't spend
much time testing what happens if upgrading from 2.x.

**Comment by ctbritt** [comment]
Awesome! Works great! Thanks!
